resolwe/test_helpers/test_runner.py

```diff
@@ -515,17 +543,41 @@ def resolve_process_tags(self, files):
         for proc_path in processes_paths:
             process_schemas.extend(self.find_schemas(proc_path))
 
+        # Switch to source branch and get all the schemas from there as well, since
+        # some schemas might have been removed.
+        with self.git_branch(self.only_changes_to):
+            for proc_path in processes_paths:
+                process_schemas.extend(self.find_schemas(proc_path))
```

Comment:
Would it make sense to check if the process schema is not already included in `process_schemas`?
It would probably often halve the size of the list.

Reply:
No, because the schema might exist in both cases but be different. So it needs to be indexed twice to cover all possible dependencies. We could check if it is the same schema that was already indexed before parsing it, but this would just complicate the logic for little gain (schema parsing is not really that expensive).

Reply:
Ok, let's leave it as is then.